py-scripts/lf_interop_rvr_test.py

```diff
@@ -476,16 +563,43 @@ def generate_individual_graphs(self, report, res, phone_x):
             if "graph_df" in phone_x:
                 phone_x.pop("graph_df")
         RSSISignal = {}
-        Throughput = {}
+        Upload = {}
+        Download = {}
```

Comment:
As the value is already there, we can change this in the future, but non-const variables should start with a lowercase letter (e.g. `my_var` vs. `MY_VAR` which is const).